packages/lodestar/src/network/reqresp/handlers/beaconBlocksByRoot.ts

```diff
@@ -1,18 +1,16 @@
-import {phase0} from "@chainsafe/lodestar-types";
+import {allForks, phase0} from "@chainsafe/lodestar-types";
 import {IBeaconChain} from "../../../chain";
 import {IBeaconDb} from "../../../db";
 
 export async function* onBeaconBlocksByRoot(
   requestBody: phase0.BeaconBlocksByRootRequest,
   chain: IBeaconChain,
   db: IBeaconDb
-): AsyncIterable<phase0.SignedBeaconBlock> {
-  const getBlock = db.block.get.bind(db.block);
-  const getFinalizedBlock = db.blockArchive.getByRoot.bind(db.blockArchive);
```

Comment:
Is there a reason why we binded the functions here instead of calling them directly?